src/dex-ui/pages/dao/ProposalDetailsPage/GovernanceProposalConfirmationDetails.tsx

```diff
@@ -129,7 +133,7 @@ export function GovernanceProposalConfirmationDetails(props: GovernanceProposalC
                 <AlertDialog
                   openDialogButtonStyles={{ flex: "1" }}
                   openDialogButtonText="Vote"
-                  isOpenDialogButtonDisabled={proposal === undefined}
+                  isOpenDialogButtonDisabled={proposal === undefined || Number(votingPower) <= 0}
```

Comment:
we can make a single const to show / hide `modal`

Reply:
Moved the logic to a constant.